app/views/active_admin/resource/_page_accordion_component_form.html.arb

```diff
@@ -18,6 +18,8 @@ html = Arbre::Context.new do
       end
       if component
         input type: 'hidden', value: component.id, name: "page[page_components_attributes][#{placeholder}][component_attributes][id]"
+
+        a 'Move to Top', href: '#', class: 'move-to-top float-right', id: component.page_component.id
       end
```

Comment:
One thing I am iffy about is the fact that the `page_components` table is not set up to use uuids meaning the id attributed to the html elements in these links are the actual ids for the component's `PageComponent` relation. Did some research and apparently its not necessarily a security risk? 

If needed I suppose the id attributed to the link can be combination of the component's class and it's uuid, pass that as the request param, then use in the controller action to look up the component in order to access it's `PageComponent` relation to make the ordering update, but I wasn't sure if its actually necessary.